scripts/build.ps1

```diff
@@ -132,6 +153,16 @@ if (-not $noVendor) {
         Download-File -Url $s.url -File $vend\$tempArchive -ErrorAction Stop
         Extract-Archive $tempArchive $s.name
 
+        # Make Embedded Windows Terminal Portable
+        if ($s.name -eq "windows-terminal") {
+          $windowTerminalFiles = resolve-path ($saveTo + "\" + $s.name + "\terminal*")
+					move-item -ErrorAction SilentlyContinue $windowTerminalFiles\* $s.name >$null
+					remove-item -ErrorAction SilentlyContinue $windowTerminalFiles >$null
```

Comment:
Can you please correct the indentation for these two lines? (Tab/Space mismatch) Sorry for the nitpicking